src/main/webapp/pages/result/resultListView.jsp

```diff
@@ -1362,14 +1362,14 @@ jQuery(document).ready(function (index) {
 							path="testResult[${iter.index}].refer"
 							onchange="toggleReferral(${iter.index});markUpdated(${iter.index});" /> <spring:message
 							code="refertest" text="Refer test to a reference lab" /></td>
-							<td width="50%"><%=MessageUtil.getMessage("workplan.method")%>&nbsp;
+							<%-- <td width="50%"><%=MessageUtil.getMessage("workplan.method")%>&nbsp;
```

Comment:
@gitcliff , i commented out this `method` related code beause it is buggy when you attempt to Search results By Patient/Oder. 
you need to fist fix it so that serach Test Results by Patient/Order doesnt fail